src/iotjs_module_process.cpp

```diff
@@ -130,7 +130,7 @@ static JResult WrapEval(const char* source) {
   int len2 = strlen(source);
   int len3 = strlen(wrapper[1]);
 
-  String code("", len1 + len2 + len3 + 1);
+  String code("", len1 + len2 + len3);
```

Comment:
@LaszloLango never thought so :) I pointed at that PR because it shows that the change can also be applied to current iot.js, and because that PR does a bit more than simply removing the +1 from the String constructor.